.github/workflows/release.yml

```diff
@@ -119,12 +184,6 @@ jobs:
         env:
           CIBW_ARCHS: ${{ matrix.arch }}
           CIBW_SKIP: pp* *musllinux* *-manylinux_i686
```

Comment:
Given this is now only mac+win, the musllinux and manylinux skips can probably be removed